docs/wiki/[User-Guide]-Starter-Module-HubNetworking.md

```diff
@@ -1,16 +1,36 @@
 <!-- markdownlint-disable first-line-h1 -->
+# Starter Module - Hub Networking
```

Comment:
For these wiki pages, I'm not sure if you are meant to include a header. See the comment above. It uses the file name as the title.